ql/src/test/queries/clientpositive/timestamp_2.q

```diff
@@ -17,6 +17,19 @@ select cast(t as float) from timestamp_2 limit 1;
 select cast(t as double) from timestamp_2 limit 1;
 select cast(t as string) from timestamp_2 limit 1;
 
+set hive.local.time.zone=Asia/Bangkok;
+
+select cast(t as boolean) from timestamp_2 limit 1;
```

Comment:
I don't see any diff between timestamp_1 and timestamp_2 testcases. Is it duplicate?

Reply:
The difference is between serde of table and the insertion data format.